### PR TITLE
{bio}[iccifort/2020.4.304] segemehl v0.3.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires #12467 (HTSlib)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2606.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/1f30db20a0c22c9e17e6a3a44d77fb55 for a full test report.

File: easybuild/easyconfigs/s/segemehl/segemehl-0.3.4-iccifort-2020.4.304.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+easyblock = 'MakeCp'
+
+name = "segemehl"
+version = "0.3.4"
+
+homepage = 'https://www.bioinf.uni-leipzig.de/Software/segemehl/'
+description = """segemehl is a software to map short sequencer reads to reference genomes. 
+ Unlike other methods, segemehl is able to detect not only mismatches but also insertions 
+ and deletions. Furthermore, segemehl is not limited to a specific read length and is able 
+ to mapprimer- or polyadenylation contaminated reads correctly. segemehl implements a matching
```
Comment (Flamefire) [inline easybuild/easyconfigs/s/segemehl/segemehl-0.3.4-iccifort-2020.4.304.eb:10]:
```suggestion
 to map primer- or polyadenylation contaminated reads correctly. segemehl implements a matching 
```
Typo in the original, fixed at e.g. https://nodepit.com/node/de.helmholtz_muenchen.ibis.ngs.segemehl.SegemehlNodeFactory

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bwd-rockylinux-88 - Linux Rocky Linux 8.8, x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), 2 x NVIDIA NVIDIA GeForce GTX 1060 6GB, 525.105.17, Python 3.6.8
See https://gist.github.com/SebastianAchilles/d28237e3fa716a9153da7f4f8eae6e5d for a full test report.

**Comment by SebastianAchilles** [comment]
Going in, thanks @boegel!

**Pull request merged**